.github/workflows/pytest-rts.yml

```diff
@@ -30,3 +30,46 @@ jobs:
         run: |
           pip install tox
           tox -e py -- --exitfirst
+
+  evaluation:
+    needs: test
+    name: Run evaluation
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone target repo
+        run: |
+          cd ../../.
+          git clone https://github.com/willmcgugan/rich.git
+      - name: Checkout
```

Comment:
I think rather then checking out the repo it would be better to install the module with:
`pip install git+ssh://git@github.com/F-Secure/pytest-rts.git#egg=pytest-rts@[HASH]`

Thus there will be less mess with folders

Reply:
and also `Install tool` step won't be needed